docs/source/faq.rst

```diff
@@ -333,6 +333,43 @@ Note that the above examples are similar to running the garbage collector inside
     :class:`~optuna.integration.ChainerMNStudy` does currently not provide ``gc_after_trial`` nor callbacks for :func:`~optuna.integration.ChainerMNStudy.optimize`.
     When using this class, you will have to call the garbage collector inside the objective function.
 
+How can I output a log only when the best value is updated?
+-----------------------------------------------------------
+
+Here's how to replace the logging feature of optuna and with your own logging callback function.
+The implemented callback can be passed to `study.optimize`.
```

Comment:
?
```suggestion
The implemented callback can be passed to ``study.optimize``.
```

Reply:
Thanks for your comment! We use double backticks for variables and single backticks for methods/functions. So, we should use a single one here.

By the way, I think it is good to use the following here.
```suggestion
The implemented callback can be passed to :func:`~optuna.study.Study.optimize`.
```

Reply:
yeah, I should have suggested `:func`'s expression. Thank you for your update.